source/labs/housekeeping/index.md

```diff
@@ -25,3 +26,231 @@ We've restored out testing environment to make sure that ported components still
 And talking about environment, we've ditched the configurtion via `config.php` and created a `.env.dist` file which now provides all credentials and configurations for the kernel. One of the best advantages when using environment variables is, that you can set them in your apache2, nginx and other webserver's config. That makes deployment a lot easier.
 
 To follow our own new concepts, we've added deptrac and other code quality tools. This prevents developers from using services in layers where they are not meant to be, like using the database connection in the controller.
+
+<div class="toc-list" data-depth="2"></div>
+
+## Database
+
+As already mentioned, the minimum required version is MySQL 5.7 and we still don't support other databases like PostgreSQL, MSSQL, ...
+
+### Conventions
+
+We've added simple conventions to keep everything straight from the beginning.
+
+- The `s_` prefix is gone
+- Every table must be singular for problems like `s_articles_details`
+- Every table and column must be lower-snake-case
```

Comment:
"lower-snake-case" is actually called [kebab-case](https://en.wikipedia.org/wiki/Letter_case#Special_case_styles) 🍌

Reply:
![image](https://user-images.githubusercontent.com/736986/32053816-ab11b59c-ba5c-11e7-89f5-94e0a9d695d9.png)
nooooope, because it still has underscores instead of hyphens :p

Reply:
Then your example string should be "lower_snake_case"....